doc/source/user/quickstart.rst

```diff
@@ -1112,8 +1112,8 @@ indices for each dimension must have the same shape.
            [[10,  9],
             [11, 11]]])
 
-Naturally, we can put ``i`` and ``j`` in a sequence (say a list) and
-then do the indexing with the list.
+In Python, ``arr[i, j]`` is exactly the same as ``arr[(i, j)]`` - so naturally, we can put ``i`` and ``j`` in a ``tuple`` and
```

Comment:
I think, but am not certain, that:
* an em-dash is appropriate here
* `---` is understood by Sphinx as an em-dash
* spaces should not be used around em-dashes
* `\ ` is needed after a ``` ``code`` ``` block if there is no space.
* this line is longer than we usually have

If those are all true, then:
```suggestion
In Python, ``arr[i, j]`` is exactly the same as ``arr[(i, j)]``\ ---so
naturally, we can put ``i`` and ``j`` in a ``tuple`` and
```

Reply:
> `\` is needed after a \``code\`` block if there is no space.
In this file it doesn't do that at the end of sentences. For example at **The Basics** section:

```
ndarray.size
    the total number of elements of the array. This is equal to the
    product of the elements of ``shape``.
```

Reply:
Good point - it definitely does matter for non-punctuation characters though.

Reply:
So it shouldn't matter for the em-dash right?